server/service/datadeal/handler_data_validation.go

```diff
@@ -150,11 +150,15 @@ func (s *dataDealService) ValidateData(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	payload := datadealtypes.Certificate{
+	certificate := &datadealtypes.Certificate{
 		UnsignedCertificate: unsignedDataCert,
 		Signature:           sig.Serialize(),
 	}
 
+	payload := datadealtypes.Consent{
+		Certificate: certificate,
+	}
+
```

Comment:
As @youngjoon-lee said in offline channel, oracle will just return the certificate, and provider will make consent to provide the data with this certificate.
So, let's just return the certificate itself

Reply:
Yes, when I tested this code and seller request validating a data, this API returns like this:
```json
{
    "certificate": {
        "unsigned_certificate": {
            "cid": "...",
            "unique_id": "...",
            "oracle_address": "...",
            "deal_id": 1,
            "provider_address": "...",
            "data_hash": "..."
        },
        "signature": "..."
    }
}
```
Because consent includes a certificate, so would I need to change it?

Reply:
Yes it includes certificate. I just wanna say that returning consent and returning certificate are distinctly different.

For example, in the future, let's assume that we add a expiration for the consent (how long the provider provide the data).
Then, the provider will get a certificate from oracle, and make consent message to provide data with expiration he/she want. Making the consent is the provider's job.

Reply:
As we discussed in offline, we will be rollback. I understand the @H4NLee's comment. And I think that it is correct.
Fixed in [d3e313e](https://github.com/medibloc/panacea-oracle/pull/67/commits/d3e313e3f72e142337ffde989b26522d03553a64).

Reply:
Good. I agree. 
Thank you!